renovate.json

```diff
@@ -6,7 +6,7 @@
   "commitMessageTopic": "{{depName}}",
   "dependencyDashboard": true,
   "rebaseWhen": "conflicted",
-  "ignoreDeps": ["jsdom"],
+  "ignoreDeps": [],
```

Comment:
@cherniavskii On the latest major they reverted a change that initially reduced the performance
https://github.com/jsdom/jsdom/releases/tag/24.0.0